src/server/scripts/Spells/spell_generic.cpp

```diff
@@ -5225,6 +5225,31 @@ class spell_gen_despawn_all_summons_owned_by_caster : public SpellScript
     }
 };
 
+// Personal resurrections in battlegrounds
+// 156758 - Spirit Heal
+class spell_gen_spirit_heal : public AuraScript
```

Comment:
Maybe we could change it to spell_gen_bg_spirit_heal? Just a small suggestion, not necessarily at all, but it's more clear what spirit heal it refers to.

Reply:
i haven't researched battlefields yet. Wintergrasp changed a bit on retail and now you can choose your graveyard location etc. Not sure which spells are being used there. Might be this one as well which wouldn't fit the bg prefix then